app/src/components/domain/RiskImminentEvents/WfpAdam/EventDetails/index.tsx

```diff
@@ -130,16 +128,16 @@ function EventDetails(props: Props) {
                     const date = new Date(track_date);
 
                     return {
-                        id: date.getTime(),
+                        id: track_date,
```

Comment:
Any reason to change `date.getTime()` to `track_date`?
Is `track_date` unix timestamp or ISO date string?